org.eclipse.Eclipse.appdata.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>org.eclipse.eclipse</id>
+  <metadata_license>CC-BY-SA-3.0</metadata_license>
+  <project_license>EPL-1.0</project_license>
```

Comment:
How about the license of bundled components? Surely the Eclipse build is using Java libraries which have their own licenses. Usually in the Java world the `Apache-2.0` license is used extensively, and maybe others. You may need to check the included components and pick the license identifiers [from the SPDX index](https://spdx.org/licenses/).